generators/server/templates/gradle.properties.ejs

```diff
@@ -20,65 +20,62 @@ rootProject.name=<%= dasherizedBaseName %>
 profile=dev
 
 # Build properties
-node_version=<%= NODE_VERSION %>
-npm_version=<%= NPM_VERSION %>
+nodeVersion=<%= NODE_VERSION %>
+npmVersion=<%= NPM_VERSION %>
 
 # Dependency versions
-jhipster_dependencies_version=<%= JHIPSTER_DEPENDENCIES_VERSION %>
+jhipsterDependenciesVersion=<%= JHIPSTER_DEPENDENCIES_VERSION %>
 # The spring-boot version should match the one managed by
-# https://mvnrepository.com/artifact/io.github.jhipster/jhipster-dependencies/${jhipster_dependencies_version}
-spring_boot_version=<%= SPRING_BOOT_VERSION %>
+# https://mvnrepository.com/artifact/io.github.jhipster/jhipster-dependencies/<%= JHIPSTER_DEPENDENCIES_VERSION %>
+springBootVersion=<%= SPRING_BOOT_VERSION %>
 # The hibernate version should match the one managed by
-# https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies/${spring-boot.version} -->
-hibernate_version=<%= HIBERNATE_VERSION %>
-mapstruct_version=1.3.1.Final
-archunit_junit5_version=0.14.1
-<%_ if (searchEngine === 'elasticsearch') { _%>
-log4j2_mock_version=0.0.2
```

Comment:
👍 I forgot to remove it during the Elasticsearch upgrade: https://github.com/jhipster/generator-jhipster/pull/11683
Could you do me a favour, and also remove it in `pom.xml.ejs` template?

Reply:
Sure :slightly_smiling_face: